src/app/components/article/Articles.js

```diff
@@ -43,20 +45,16 @@ const adjustFilters = (filters) => {
     delete newFilters.language;
   }
 
-  // Some aliases
   if (filters.users) {
     newFilters.user_ids = filters.users;
-    delete newFilters.users;
```

Comment:
This was to fix the bug found by Jessie in CV2-5988. We actually need these keys for the filters to display in the UI